### DISCUSSION TITLE
Filter numbers were internationalized

### DISCUSSION THREAD

**Discussion by angrybrad** [craftcms/cms#9484] [Ideas]
> Created by: Knut (knut.svangstu@gmail.com) on 2015/10/09 08:04:22 +0000
> Votes at time of UserVoice import: 2

-----------------------------------

The "currency" and "filesize" filters are unusable if you make websites in most european languages where dot or small space separates thousands and comma separates whole numbers from decimals (1.000,00 or 1 000,00). Many languages also handle currency outputs slightly differently; in Norway it's common to write 100,- instead of 100,00 while in Sweden it's written 100;-

Another pet pewe with currency is that for Norwegian Crowns the field outputs Nok 100.00. "NOK" is only used when different crown-based currencies are mixed on a single page (NOK, SEK, DKK), otherwise the currency is always written "kr" and placed after, not before the number.

**Comment by missmatsuko** [comment]
I also find the `filesize` filter kind of weird to use in practice. It's fine if you want really precise base-10 conversions, but I think the average user just wants a general idea of a file's size.

I got `11.802 MiB` from `12375346`, but if I'm displaying it on the front-end to the average user, I would want it to be less precise and suffixed with MB rather than MiB. So `11.8 MB` or even `12MB` rather than `11.802 MiB`. It would be nice to have options for rounding and binary vs base-10 units.
